### DISCUSSION TITLE
Going all in and choosing good cameras [SOLVED by moandcompany]

### DISCUSSION THREAD

**Discussion by digitalit** [blakeblackshear/frigate#3822] [Ask A Question] [answered]
Hi guys,

I have tryed a lot of solutions but ended up with an pretty powerfull server running unRaid and Hassos in a VM. In Hassos i run the Frigate HACS and everything is working great. I have a Google Coral as recommended but my cameras are a bit old an i get quite a few errors about streams and what not so now i will buy the "correct" and recommended cameras for Frigate.

This one was recommended on the official site: https://www.amazon.com/gp/product/B07S5QZJDH/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B07S5RM7JG&linkCode=as2&tag=frigate0d-20&linkId=c0a5da91cc408e157739a7aa60d157cd&th=1

In Sweden i can buy this model and i wonder if this model has any downsides tecnically (not the price) than the one recommended: https://www.dustin.se/product/5011276895/wizsense-5mp-ip67-dome-camera?tab=specification

Thank you so much and have a nice day

**Comment by moandcompany** [comment, marked as answer]
Regarding technical downsides:

1) The recommended camera here, and commonly elsewhere is the Dahua 5442, which is a 4MP camera using a 1/1.8" Sensor Size
2) Second camera referenced is a [Dahua 3549](https://www.dahuasecurity.com/products/productDetail/43547), which is a 5MP camera using 1/2.5" Sensor Size

The Dahua 3549 has a physically smaller image sensor than the Dahua 5442

When looking at the technical specifications of these two cameras, the Sensor Size is very meaningful because it provides us information to understand the pixel density of the image/video sensors and how much light is available for each pixel.

Generally, when assuming identical exposure settings (e.g. aperture/iris size, exposure time, gain), the camera with the lower pixel density will perform better in low ambient light conditions (i.e. the lower pixel density camera will be expected to have a cleaner image signal, and more usable, image with less noise than the higher pixel density camera).

If your camera application is in an environment with lots of ambient lighting (e.g. outdoor, daytime application), these specifications will matter less, however if your application is in an area with little ambient light, the Dahua 5442 will probably be more suited to the task.

**Comment by rhatguy** [comment]
I can say I have had good luck with the Loryta cams recommended by frigate after having horrible luck with Reolinks previously.  I run one T2431and 4 other T5442 (recommended cam).  I can say its night and day different having used the reolinks before.  These Loryta (rebranded Dahua) cams streams are SOLID and quick.  I'd HIGHLY recommend these cams myself.

**Comment by digitalit** [comment]
@moandcompany and @rhatguy , thank you so much for helping me figure this out.

Didn't expect this educational and easy understandable answers. 

Were we live now and based on @moandcompany great words i guess the Dahua 5442 is more suitable so i now have placed an order for 3 cameras.

As i said i have (in my world) a quite powerfull server and i can give my Hassos VM a lot more power but looking at for example Frigates CPU and RAM usage i feel that the VM has enough power.

Please have a look at this picture and tell me if this looks okay:
![frigate](https://user-images.githubusercontent.com/6569136/190095685-60e7bfed-94f2-45dc-8292-fef6b77d6084.png)


And i´m sorry to ask since this question is a bit off Frigate topic and more an unRaid question but can you please tell me if this looks okay for my VM:
![hassos](https://user-images.githubusercontent.com/6569136/190095957-5e6bb832-d549-4b2f-9f77-b775fffe5331.png)

Again, thank you so much for this help and have a nice day.
